internal/services/media/media_transform_resource.go

```diff
@@ -188,14 +225,765 @@ func resourceMediaTransform() *pluginsdk.Resource {
+										ValidateFunc: validation.StringInSlice(encodings.PossibleValuesForBlurType(), false),
+									},
+									"experimental_options": {
+										Type:     pluginsdk.TypeMap,
+										Optional: true,
+										Elem: &pluginsdk.Schema{
+											Type: pluginsdk.TypeString,
+										},
+									},
+									"face_redactor_mode": {
+										Type:         pluginsdk.TypeString,
+										Optional:     true,
+										Default:      string(encodings.FaceRedactorModeAnalyze),
+										ValidateFunc: validation.StringInSlice(encodings.PossibleValuesForFaceRedactorMode(), false),
+									},
+								},
+							},
+						},
+						"custom_preset": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &pluginsdk.Resource{
+								Schema: map[string]*pluginsdk.Schema{
+									"codec": {
+										Type:     pluginsdk.TypeList,
+										Required: true,
+										MinItems: 1,
+										Elem: &pluginsdk.Resource{
+											Schema: map[string]*schema.Schema{
+												"aac_audio": {
+													Type:     pluginsdk.TypeList,
+													Optional: true,
+													MaxItems: 1,
+													Elem: &pluginsdk.Resource{
+														Schema: map[string]*schema.Schema{
+															"bitrate": {
+																Type:         pluginsdk.TypeInt,
+																Optional:     true,
+																Default:      128000,
+																ValidateFunc: validation.IntAtLeast(1),
+															},
+															"channels": {
+																Type:         pluginsdk.TypeInt,
+																Optional:     true,
+																Default:      2,
+																ValidateFunc: validation.IntBetween(1, 6),
+															},
+															"label": {
+																Type:         pluginsdk.TypeString,
+																Optional:     true,
+																ValidateFunc: validation.StringIsNotEmpty,
+															},
+															"profile": {
+																Type:         pluginsdk.TypeString,
+																Optional:     true,
+																Default:      string(encodings.AacAudioProfileAacLc),
+																ValidateFunc: validation.StringInSlice(encodings.PossibleValuesForAacAudioProfile(), false),
+															},
+															"sampling_rate": {
+																Type:         pluginsdk.TypeInt,
+																Optional:     true,
+																Default:      48000,
+																ValidateFunc: validation.IntBetween(11025, 96000),
+															},
+														},
+													},
+												},
+												"copy_audio": {
+													Type:     pluginsdk.TypeList,
+													Optional: true,
+													MaxItems: 1,
+													Elem: &pluginsdk.Resource{
+														Schema: map[string]*schema.Schema{
+															"label": {
+																Type:         pluginsdk.TypeString,
+																Optional:     true,
+																ValidateFunc: validation.StringIsNotEmpty,
+															},
+														},
+													},
+												},
```

Comment:
does it make sens to have a block with a single value? this might be better as copy_audio_label?

Reply:
can we keep this as a nested block, to be consistent with other nested blocks at the same level, which all have a `label` property.

Reply:
sure